agent/engine/engine_unix_integ_test.go

```diff
@@ -1526,3 +1528,69 @@ func TestFluentdTag(t *testing.T) {
 	err = utils.SearchStrInDir(logdir, "ecsfts", logTag)
 	require.NoError(t, err, "failed to find the log tag specified in the task definition")
 }
+
+func TestDockerExecAPI(t *testing.T) {
+	testTimeout := 1 * time.Minute
+	taskEngine, done, _ := setupWithDefaultConfig(t)
+	defer done()
+
+	stateChangeEvents := taskEngine.StateChangeEvents()
+
+	taskArn := "testDockerExec"
+	testTask := createTestTask(taskArn)
+
+	A := createTestContainerWithImageAndName(baseImageForOS, "A")
+
+	A.EntryPoint = &entryPointForOS
+	A.Command = []string{"sleep 30"}
```

Comment:
Shouldn't his be: `[]string{"sleep","30"}` ?

Reply:
Fixed it. Thanks for pointing

Reply:
Since I am using entrypoint "sh, -c", that is why "sleep 30" works fine.

Reply:
Since I am using entrypoint "sh, -c", the command "sleep 30" works fine without the comma